documentation/modules/ROOT/pages/transformations/vitess-remove-field.adoc

```diff
@@ -0,0 +1,84 @@
+:page-aliases: configuration/vitess-remove-field.adoc
+[id="vitess-remove-field"]
+= Vitess Remove Field
+
+:toc:
+:toc-placement: macro
+:linkattrs:
+:icons: font
+:source-highlighter: highlight.js
+
+toc::[]
+
+The vitess-specific `RemoveField` SMT is used to transform the records by removing a specific field given the path to that field. This can be used with the Vitess connector to remove the transaction.id field when transaction metadata is enabled to avoid storing a duplicate entry of the VGTID.
+
+It is done by using the `field_names` config which is a csv of the paths of the fields to remove. For each field specified it is removed from the record and the schema is updated as well.
+
```

Comment:
```suggestion
The following example shows how to add the SMT to the connector configuration so that it removes specific fields from the change event records that the connector emits.
```